src/core_modules/capture-core/rules/converters/dateUtils.js

```diff
@@ -47,6 +47,7 @@ class DateUtils implements IDateUtils {
         return `'${newRulesDate}'`;
     }
     compareDates(firstRulesDate: string, secondRulesDate: string): number {
+        secondRulesDate = moment().toISOString();
```

Comment:
Why is `secondRulesDate` value overwritten here? Maybe can be removed from the functions parameters `compareDates(firstRulesDate: string, secondRulesDate: string)` 
Thanks!

Reply:
Good catch! I was thinking about refactoring it, but didn't complete the job. Reverting the change here.